### ISSUE TITLE
How do I fix this? 

### ISSUE THREAD

**Issue by TsukiZero** [LagoLunatic/DSVEdit#89]
![image](https://github.com/user-attachments/assets/0bf62468-6cb4-43bc-a56e-6a466f4986ca)

**Comment by TsukiZero** [comment]
![image](https://github.com/user-attachments/assets/57d1bd64-b4b3-42b4-bf39-c22d2120daa2)

**Comment by piratesephiroth** [comment]
this is why you should make backups often

**Comment by TsukiZero** [comment]
yeah, also it's possible that I deleted a txt by mistake (I had a few other TXT aside of the file DSVEdit makes.

Ideally the DSVedit TXT file in the folder should be extension-less or have an unusual extension to prevent it being accidentally tampered with.

And perhaps it would be a good idea to have an additional folder to keep stuff in uncompressed state for GBA games, so in case disaster strikes it should be quicker to recover from.
